lib/dor/services/workflow_service.rb

```diff
@@ -540,8 +547,12 @@ def workflow_resource_method(uri_string, meth = 'get', payload = '', opts = {})
 
           response.body
         end
-      end
+      rescue *workflow_service_exceptions_to_catch => e
+        response = e.response if e.respond_to? :response
+        response ||= {}
```

Comment:
Not opposed to this pattern, esp where long lines are a problem, but maybe:
`response = e.respond_to?(:response) ? e.response : {}`
Or, despite rubocop complaints:
`response = e.response rescue {}`